optimum/exporters/tasks.py

```diff
@@ -217,6 +217,7 @@ class TasksManager:
             "multiple-choice": "AutoModelForMultipleChoice",
             "object-detection": "AutoModelForObjectDetection",
             "question-answering": "AutoModelForQuestionAnswering",
+            "reinforcement-learning": "AutoModel",
```

Comment:
this is not the right class, this is the feature extraction task class, and it's probably why you're getting nans as output

Reply:
this model should rather be registered with custom architectures, since it doesn't have an auto class family.

Reply:
OK, thanks for the hint. This could be the reason.

Reply:
oh you're right ! thanks for investigating !

Reply:
going through the hub, I see that the reinforcement task being attribute to multiple auto models, eg https://huggingface.co/jat-project/jat is used with the AutoModelForCausalLM, so let's make it a tuple, with ("AutoModel",)